esp/services/ws_logaccess/WsLogAccessService.cpp

```diff
@@ -13,11 +13,85 @@ bool Cws_logaccessEx::onGetLogAccessInfo(IEspContext &context, IEspGetLogAccessI
     bool success = true;
     if (queryRemoteLogAccessor())
     {
+        IPropertyTree * logColumnDefinitions = queryRemoteLogAccessor()->queryLogMap();
+        if (!logColumnDefinitions)
+        {
+            success = false;
+            OERRLOG("WsLogaccess: NULL col defintions - check logaccess logmap config!");
+        }
+        else
+        {
+            double clientVer = context.getClientVersion();
+
+            if (clientVer >= 1.04)
+            {
+                IArrayOf<IEspLogColumn> logColumns;
+                Owned<IPropertyTreeIterator> columnIter = logColumnDefinitions->getElements("logMaps");
+                ForEach(*columnIter)
+                {
+                    IPropertyTree & column = columnIter->query();
```

Comment:
trivial: to be consistent with line 11, remove the space from the front of the '*'.